trade-finance-manager-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -0,0 +1,35 @@
+const { param } = require('express-validator');
+
+const userParamNumericValidation = param('user').isAlphanumeric().withMessage('The User ID (user) provided should be numeric');
+const facilityIdNumericValidation = param('facilityId').isNumeric().withMessage('The Facility ID (facilityId) provided should be numeric');
+const amendmentIdNumericValidation = param('amendmentId').isNumeric().withMessage('The Amendment ID (amendmentId) provided should be numeric');
+const dealIdNumericValidation = param('dealId').isNumeric().withMessage('The Amendment ID (amendmentId) provided should be numeric');
+const partyURNNumericValidation = param('urn').isNumeric().withMessage('The party URN (urn) provided should be numeric');
+
+exports.userEndpointValidation = [userParamNumericValidation];
+
```

Comment:
some of these validations will repeat themselves (i.e. the facilityId will be validated on multiple endpoints).
I wonder if it's worth combining them into 1 statement, rather than having multiple statements. i.e. when it comes to validating the facilityId, we can use one statement:
```js
exports.facilityIdValidation = [facilityIdNumericValidation];
```

Reply:
Yep wasn't sure on my preference between per endpoint or reducing repetition as much as possible - happy to refactor to above